openapi/openapi.yaml

```diff
@@ -308,6 +315,14 @@ paths:
     $ref: 'services/harbour.yaml#/harbour-user'
   /harbour/auth/classic:
     $ref: 'services/harbour.yaml#/harbour-auth-classic'
+  /journals/Summary/{bibstem}:
```

Comment:
Our URLs are generally (always?) stylized in all-lowercase - is there a need for these to be capitalized? If not, I'd change them for consistency

Reply:
This has been modified in the documentation.  I have a PR in progress to change the endpoints in ADSJournalsDB/journalsservice/app.py to reflect this.